static/app/components/events/contexts/utils.tsx

```diff
@@ -27,102 +26,37 @@ import type {AvatarUser} from 'sentry/types/user';
 import {defined} from 'sentry/utils';
 import commonTheme from 'sentry/utils/theme';
 
-import {AppEventContext, getKnownAppContextData, getUnknownAppContextData} from './app';
-import {
-  BrowserEventContext,
-  getKnownBrowserContextData,
-  getUnknownBrowserContextData,
-} from './browser';
-import {DefaultContext, getDefaultContextData} from './default';
-import {
-  DeviceEventContext,
-  getKnownDeviceContextData,
-  getUnknownDeviceContextData,
-} from './device';
-import {getKnownGpuContextData, getUnknownGpuContextData, GPUEventContext} from './gpu';
+import {getKnownAppContextData, getUnknownAppContextData} from './app';
+import {getKnownBrowserContextData, getUnknownBrowserContextData} from './browser';
+import {getDefaultContextData} from './default';
+import {getKnownDeviceContextData, getUnknownDeviceContextData} from './device';
+import {getKnownGpuContextData, getUnknownGpuContextData} from './gpu';
 import {
   getKnownMemoryInfoContextData,
   getUnknownMemoryInfoContextData,
-  MemoryInfoEventContext,
 } from './memoryInfo';
 import {
   getKnownOperatingSystemContextData,
   getUnknownOperatingSystemContextData,
-  OperatingSystemEventContext,
 } from './operatingSystem';
 import {
   getKnownPlatformContextData,
   getPlatformContextIcon,
   getUnknownPlatformContextData,
   KNOWN_PLATFORM_CONTEXTS,
 } from './platform';
-import {
-  getKnownProfileContextData,
-  getUnknownProfileContextData,
-  ProfileEventContext,
-} from './profile';
-import {getReduxContextData, ReduxContext} from './redux';
-import {
-  getKnownReplayContextData,
-  getUnknownReplayContextData,
-  ReplayEventContext,
-} from './replay';
-import {
-  getKnownRuntimeContextData,
-  getUnknownRuntimeContextData,
-  RuntimeEventContext,
-} from './runtime';
-import {
-  getKnownStateContextData,
-  getUnknownStateContextData,
-  StateEventContext,
-} from './state';
+import {getKnownProfileContextData, getUnknownProfileContextData} from './profile';
+import {getReduxContextData} from './redux';
+import {getKnownReplayContextData, getUnknownReplayContextData} from './replay';
+import {getKnownRuntimeContextData, getUnknownRuntimeContextData} from './runtime';
+import {getKnownStateContextData, getUnknownStateContextData} from './state';
 import {
   getKnownThreadPoolInfoContextData,
   getUnknownThreadPoolInfoContextData,
-  ThreadPoolInfoEventContext,
 } from './threadPoolInfo';
-import {
-  getKnownTraceContextData,
-  getUnknownTraceContextData,
-  TraceEventContext,
-} from './trace';
-import {
-  getKnownUnityContextData,
-  getUnknownUnityContextData,
-  UnityEventContext,
-} from './unity';
-import {
-  getKnownUserContextData,
-  getUnknownUserContextData,
-  UserEventContext,
```

Comment:
And this whole concept of 'known' and 'unknown' keys and having a function for each is completely gone (see `cloud_resource`, `culture` and `missing_instrumentation` contexts). That is even more code I can delete, but even more that i'll save for follow up PRs. this is my life now